typings/node/node.d.ts

```diff
@@ -291,7 +292,7 @@ declare module "http" {
         setEncoding(encoding?: string): void;
         pause(): void;
         resume(): void;
-        connection: net.Socket;
+        connection: net.Socket|tls.ClearTextStream;
```

Comment:
This is a local change for now and is a pending PR for borisyankov/DefinitelyTyped#3517.